backends/dpdk/DpdkXfail.cmake

```diff
@@ -96,6 +92,11 @@ p4c_add_xfail_reason("dpdk"
   testdata/p4_16_samples/pna-example-mirror-packet-error3.p4
   )
 
+p4c_add_xfail_reason("dpdk"
```

Comment:
Why this failure? Is this an error case?

Reply:
parser block does is not allowing to acces input port, I think it's a bug in compiler, i will create jira for it.